samples/bluetooth/mesh/light_switch/lpn.conf

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_MESH_LOW_POWER=y
+CONFIG_BT_DEVICE_NAME="Mesh Light Switch LPN"
+CONFIG_BT_MESH_RELAY=n
+CONFIG_BT_MESH_FRIEND=n
+
+# Power saving configuration:
+
+# Disable serial communication
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+
+# Disable secure beacons
+CONFIG_BT_MESH_BEACON_ENABLED=n
+
+# Extend LPN poll interval to 1 minute
+CONFIG_BT_MESH_LPN_POLL_TIMEOUT=600
```

Comment:
I have investigated it a bit. Sure, we could add it to emphasize that lowering this parameter can reduce power consumption in general, but it has no impact on the consumption in the sample since Node ID is never advertised. I tried tinkering with `bt_mesh_subnet_node_id_set` like suggested in the discussion on this task, but I was not able to get rid of the advertisment with net idx. Instead I ended up using `bt_mesh_gatt_proxy_set` instead to achive it.